test/records/10df2a6d7adc190767b250ec0fea7d94c00fef6c.json

```diff
@@ -1 +1 @@
,\n \"parents\": [\n  {\n   \"kind\": \"drive#parentReference\",\n   \"id\": \"0AIiujYcoBWhwUk9PVA\",\n   \"selfLink\": \"https://www.googleapis.com/drive/v2/files/1hI03GihT5CAAxAAHjwKEqNPlDotK55ywdNTmAKrrfFs/parents/0AIiujYcoBWhwUk9PVA\",\n   \"parentLink\": \"https://www.googleapis.com/drive/v2/files/0AIiujYcoBWhwUk9PVA\",\n   \"isRoot\": true\n  }\n ],\n \"exportLinks\": {\n  \"application/x-vnd.oasis.opendocument.spreadsheet\": \"https://docs.google.com/spreadsheets/export?id=1hI03GihT5CAAxAAHjwKEqNPlDotK55ywdNTmAKrrfFs&exportFormat=ods\",\n  \"text/tab-separated-values\": \"https://docs.google.com/spreadsheets/export?id=1hI03GihT5CAAxAAHjwKEqNPlDotK55ywdNTmAKrrfFs&exportFormat=tsv\",\n  \"application/pdf\": \"https://docs.google.com/spreadsheets/export?id=1hI03GihT5CAAxAAHjwKEqNPlDotK55ywdNTmAKrrfFs&exportFormat=pdf\",\n  \"application/vnd.openxmlformats-officedocument.spreadsheetml.sheet\": \"https://docs.google.com/spreadsheets/export?id=1hI03GihT5CAAxAAHjwKEqNPlDotK55ywdNTmAKrrfFs&exportFormat=xlsx\",\n  \"text/csv\": \"https://docs.google.com/spreadsheets/export?id=1hI03GihT5CAAxAAHjwKEqNPlDotK55ywdNTmAKrrfFs&exportFormat=csv\",\n  \"application/zip\": \"https://docs.google.com/spreadsheets/export?id=1hI03GihT5CAAxAAHjwKEqNPlDotK55ywdNTmAKrrfFs&exportFormat=zip\",\n  \"application/vnd.oasis.opendocument.spreadsheet\": \"https://docs.google.com/spreadsheets/export?id=1hI03GihT5CAAxAAHjwKEqNPlDotK55ywdNTmAKrrfFs&exportFormat=ods\"\n },\n \"userPermission\": {\n  \"kind\": \"drive#permission\",\n  \"etag\": \"\\\"08z8BR1n-DRPUqokV3fOX3Mqkto/ZDP3m_JyGE4NZ3ZbPIFSbvpv5fg\\\"\",\n  \"id\": \"me\",\n  \"selfLink\": \"https://www.googleapis.com/drive/v2/files/1hI03GihT5CAAxAAHjwKEqNPlDotK55ywdNTmAKrrfFs/permissions/me\",\n  \"role\": \"owner\",\n  \"type\": \"user\"\n },\n \"quotaBytesUsed\": \"0\",\n \"ownerNames\": [\n  \"unittest-collection-write@hyou-integration-testing.iam.gserviceaccount.com\"\n ],\n \"owners\": [\n  {\n   \"kind\": \"drive#user\",\n   \"displayName\": \"unittest-collection-write@hyou-integration-testing.iam.gserviceaccount.com\",\n   \"isAuthenticatedUser\": true,\n   \"permissionId\": \"09492800149190184152\",\n   \"emailAddress\": \"unittest-collection-write@hyou-integration-testing.iam.gserviceaccount.com\"\n  }\n ],\n \"lastModifyingUserName\": \"unittest-collection-write@hyou-integration-testing.iam.gserviceaccount.com\",\n \"lastModifyingUser\": {\n  \"kind\": \"drive#user\",\n  \"displayName\": \"unittest-collection-write@hyou-integration-testing.iam.gserviceaccount.com\",\n  \"isAuthenticatedUser\": true,\n  \"permissionId\": \"09492800149190184152\",\n  \"emailAddress\": \"unittest-collection-write@hyou-integration-testing.iam.gserviceaccount.com\"\n },\n \"capabilities\": {\n  \"canCopy\": true,\n  \"canEdit\": true\n },\n \"editable\": true,\n \"copyable\": true,\n \"writersCanShare\": true,\n \"shared\": false,\n \"explicitlyTrashed\": false,\n \"appDataContents\": false,\n \"spaces\": [\n  \"drive\"\n ]\n}\n"}
```

Comment:
For my understanding, what's the deal with all these json files with uuid names? Are they captures of API operations saved for testing?

Reply:
Exactly, basically there's a "record" mode that you can use to run the tests (with the exception of `integration_test.py`) and in which all API responses are saved to disk. For future runs, the responses are played back so that the tests don't actually send any requests over the network. It's a neat way of really isolating the functionality to be tested, but a bit of a chore if you want to add new tests..